web_external/stylesheets/body/featuresetPage.styl

```diff
@@ -1,4 +1,40 @@
-.isic-featureset-global-features-content
-.isic-featureset-local-features-content
-  display inline-block
-  width 800px
+.isic-featureset-container
+
+  .isic-featureset-table
+    width 100%
```

Comment:
I don't think it looks good to have empty table stretching so far to the right, but maybe the solution is to make all of the `listingPage` occupy less than the full width of the page (which would better match the front page).